config/manager/manager.yaml

```diff
@@ -102,5 +104,14 @@ spec:
           requests:
             cpu: 10m
             memory: 64Mi
+        volumeMounts:
+          - name: webhook-certs
+            mountPath: /tmp/k8s-webhook-server/serving-certs
+            readOnly: true
       serviceAccountName: controller-manager
       terminationGracePeriodSeconds: 10
+      volumes:
+        - name: webhook-certs
+          secret:
+            secretName: webhook-server-cert
+            optional: true
```

Comment:
Do we need those changes here? I think `config/default/manager_webhook_patch.yaml` will do that.